packages/store/index.js

```diff
@@ -8,6 +8,15 @@ const addonBaseConfig = addonBuildConfigForDataPackage(name);
 
 module.exports = Object.assign({}, addonBaseConfig, {
   shouldRollupPrivate: true,
+
+  included() {
```

Comment:
note for myself: before merging check this works correctly given what we do in addonBaseConfig